src/Auth/EnvMomentoTokenProvider.php

```diff
@@ -19,10 +19,10 @@ public function __construct(
         ?string $trustedCacheEndpointCertificateName = null
     )
     {
-        $authToken = getenv($envVariableName);
-        if ($authToken === false || isNullOrEmpty($authToken)) {
+        if (! array_key_exists($envVariableName, $_SERVER)) {
```

Comment:
array key exists is not correct to use here, since it will pass if the value is present but null?